packages/components/src/Combobox/components/Content/Content.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import classnames from "classnames";
+import styles from "./Content.css";
+import { Icon } from "../../../Icon";
+import { ComboboxContext } from "../../ComboboxProvider";
+
+interface ContentProps {
+  /**
+   * options for the combobox
+   */
+  readonly options: string[];
+
+  /**
+   * onSelection handler.
+   */
+  onSelection: (selection: string) => void;
+
+  /**
+   * actions
+   */
+  readonly children: React.ReactNode;
```

Comment:
if this isn't gonna accept a string and just a list of elements 

```suggestion
  readonly children: ReactElement | ReactElement[];
```